Tests/WebTest/SitemapTest.php

```diff
@@ -87,14 +90,19 @@ public function getFormats()
                          <xhtml:link href="http://localhost/sitemap-aware?_locale=de" hreflang="de" rel="alternate"/>
                      </url>
                      <url>
+                         <loc>http://localhost/sitemap-aware-last-mod-date</loc>
+                         <lastmod>2016-07-06T00:00:00+02:00</lastmod>
```

Comment:
maybe you can explicitly set the timezone to Europe/Berlin in your
template? or in the autoload/bootstrap file?

surprised that this did not turn up on travis-ci. but maybe they are
dutch and thus in the same timezone as us...